translations/texts/objects/apex/apexarmchair/apexarmchair.object.json

```
@@ -92,7 +92,7 @@
     },
     "Texts": {
       "Eng": "This upholstery is bananas.",
-      "Rus": ""
+      "Rus": "Эта обивка бананов."
```

```suggestion
      "Rus": "Эта обивка состоит из бананов."
```
Никто бананы не обивал. Это обивка кресла, но дословно она - бананы.